ext/mysql2/result.h

```diff
@@ -1,5 +1,6 @@
 #ifndef MYSQL2_RESULT_H
 #define MYSQL2_RESULT_H
+#include <stdbool.h>
```

Comment:
I remember reading at one point that Ruby wants extensions to compile with C89? Is that still true?